### PR TITLE
SOM machinegun

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Adds the V-41 general machine gun to the SOM arsenal.
Its basically a MG-60 equivilent, but with slightly strong damage per bullet at the cost of lower ROF and bullets per mag.

Shout out to Crow for the great sprites.

image

It's not a spectacular gun or anything, but adds more variety for the SOM, and in particular for sensor cap may come in handy for defence.

## Why It's Good For The Game
More SOM dakka good

## Changelog
:cl:
add: Added the V-41 machine gun for the SOM
/:cl:

### CODE REVIEW

**Comment by Mannybrado** [comment]
ADD AIM MODE FOR IT PLEASE THANKS :D

File: code/datums/quick_load_outfits.dm [hunk 1]
```diff
@@ -1366,6 +1366,41 @@
 	H.equip_to_slot_or_del(new /obj/item/explosive/grenade/incendiary, SLOT_IN_ACCESSORY)
 	H.equip_to_slot_or_del(new /obj/item/explosive/grenade/flashbang/stun, SLOT_IN_ACCESSORY)
 
+/datum/outfit/quick/som/marine/machine_gunner
+	name = "V-41 Rifleman"
```
Comment (XSlayer300) [inline code/datums/quick_load_outfits.dm:1370]:
```suggestion
	name = "V-41 Machinegunner"
```

**Comment by XSlayer300** [comment]
also, V-41 guys do not have mesons or something, intentional?

**Comment by Lumipharon** [comment]
> also, V-41 guys do not have mesons or something, intentional?

Yeah, just the vets (and actual engies) that have em.

**Comment by Lumipharon** [comment]
> ADD AIM MODE FOR IT PLEASE THANKS :D

It has it yes

File: code/modules/projectiles/gun_attachables.dm [hunk 2]
```diff
@@ -1022,6 +1022,13 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	name = "AR-11 stock"
 	icon_state = "tx11stock"
 
+/obj/item/attachable/stock/v41stock
```
Comment (TiviPlus) [inline code/modules/projectiles/gun_attachables.dm:1025]: no numbers in typepaths

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/gun_attachables.dm (hunk 2 above)
Comment (Lumipharon) [reply code/modules/projectiles/gun_attachables.dm]: whoops

**Pull request merged**